containers/Ideas/IdeaCreateDescription.js

```diff
@@ -60,6 +71,7 @@ export const IdeaCreateDescription = props => {
                 touched={touched.description}
                 numberOfLines={10}
                 multiline={true}
+                autoFocus
               />
             </ScrollView>
```

Comment:
@phillimorland just figured out that I cannot put KeyboardAvoidingView and ScrollView into one component, because between those wrappers, we have this ButtonSubmit in this case

Reply:
@khamui as we will always use this for inputs can we not add that to the component? the point is to get the same behaviour on all text submit screens to it kind of makes sense unless i'm missing something?

Reply:
uff yeah make sense, sorry my brain is lagging today